src/strategies/taraxa-delegation/index.ts

```diff
@@ -0,0 +1,112 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { Contract } from '@ethersproject/contracts';
+import { formatUnits } from '@ethersproject/units';
+
+export const author = 'taraxa';
+export const version = '0.1.0';
+export const dependOnOtherAddress = true;
```

Comment:
Hey @VargaElod23 Right now we are not accepting any strategies that depend on other voters addresses https://github.com/snapshot-labs/snapshot-strategies#code

Reply:
Right, fixed.